styles.css

```diff
@@ -49,6 +84,27 @@
   align-items: flex-end;
   justify-content: center;
   height: 40px;
+  justify-self: flex-end;
+}
+
+#top-menu {
+  display: none;
+  margin-top: 12px;
+}
+
+.vis {
+  display: none;
+}
+
+@media only screen and (min-width: 767px) {
+  #top-menu {
```

Comment:
- With regards to the project requiremrent shown below you are expected to use `768px` not `767px` for desktop verison. please kindly make this minor change in everywhere you have used it :+1: 
- project requiurement
![768](https://user-images.githubusercontent.com/89055502/236409015-10dbc835-5f26-41e9-880e-dae506240141.png)